src/main/webapp/WEB-INF/jsp/addTopic.jsp

```diff
@@ -113,26 +113,26 @@
                         </div>
                     </div>
                 </div>
-                <div class="form-group">
+                <div class="form-group" id="radio_group">
                     <label for="subscriptionMethod" class="col-sm-3 control-label">
 												<spring:message code="addTopic.submethod"/>
 										</label>
-                    <div class="col-sm-9">
+                    <div class="col-sm-9" aria-labelledby="radio_group" aria-required="true" role="radiogroup">
                         <div class="radio">
                             <label for="subscriptionMethod1" title="<spring:message code="addTopic.pushedforced.title"/>">
-                                <form:radiobutton path="subscriptionMethod" value="1"/>
+                                <form:radiobutton role="radio" path="subscriptionMethod" value="1"/>
```

Comment:
I think role is used in a lot of assistive technology, as chromevox does not seem to know how large the set of radio buttons is without the `role` tag. It actually passed ADA before without them, but they definitely aren't completely redundant. I do not think the role tags should be on the labels because it seems to make the labels focusable on their own and confuse the location and number of the buttons. I think each radio button should only have a single focusable point

Also, a concern that I have with the code right now is that the group label cannot point to the multiple radio buttons and so the label doesn't work.